packages/pointer_interceptor/pointer_interceptor_ios/pubspec.yaml

```diff
@@ -0,0 +1,31 @@
+name: pointer_interceptor_ios
+description: iOS implementation of the pointer_interceptor plugin.
+repository: https://github.com/flutter/packages/tree/main/packages/pointer_interceptor/pointer_interceptor_ios
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3Apointer_interceptor_ios
```

Comment:
No `_ios` in the label; we don't distinguish the sub-packages in the issue labels.